CHANGELOG.md

```diff
@@ -4,7 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
-Fix: Increase line-height on tiny `Label`.
+- Fix: Add classes to resize `TextInputWithUnit`.
+- Fix: Add class to resize `FormAmountAndCurrency`.
```

Comment:
Désolé je me suis mal exprimé. Je pensais plutôt ajouter ici quelque chose comme "- Feature: Add `digits` prop to `TextInput`>